src/cluster/src/check.rs

```diff
@@ -346,7 +346,7 @@ pub(crate) fn check_helm_version(
                 required: required.to_string(),
             },
             format!(
-                "Please upgrade your helm package manager to at least {}",
+                "Please upgrade your helm client to atleast {}",
```

Comment:
typo

Reply:
I have fixed this earlier. Could you please suggest hat change is needed?

Reply:
Please upgrade your helm client to at least {}